internal/provider/datasource_directory_user_test.go

```diff
@@ -47,15 +47,15 @@ func TestDataSourceDirectoryUser(t *testing.T) {
 			ProtoV6ProviderFactories: getProviders(rec.GetDefaultClient()),
 			Steps: []resource.TestStep{
 				{
-					Config: hclProvider() + hclDatasourceDirectoryUserCustomIdp("uut", "05368777-4934-41e8-9f3c-6ec5f4d564b9", "jenny.doe@test.com", "terraformint"),
+					Config: hclProvider() + hclDatasourceDirectoryUserCustomIdp("uut", "05368777-4934-41e8-9f3c-6ec5f4d564b9", "jenny.doe@test.com", "terraformint-platform"),
 					Check: resource.ComposeAggregateTestCheckFunc(
 						resource.TestCheckResourceAttr("data.btp_directory_user.uut", "directory_id", "05368777-4934-41e8-9f3c-6ec5f4d564b9"),
 						resource.TestCheckResourceAttr("data.btp_directory_user.uut", "user_name", "jenny.doe@test.com"),
-						resource.TestCheckResourceAttr("data.btp_directory_user.uut", "origin", "terraformint"),
-						resource.TestCheckResourceAttr("data.btp_directory_user.uut", "active", "false"),
-						resource.TestCheckResourceAttr("data.btp_directory_user.uut", "family_name", ""),
-						resource.TestCheckResourceAttr("data.btp_directory_user.uut", "given_name", ""),
-						resource.TestCheckResourceAttr("data.btp_directory_user.uut", "id", ""),
+						resource.TestCheckResourceAttr("data.btp_directory_user.uut", "origin", "terraformint-platform"),
+						resource.TestCheckResourceAttr("data.btp_directory_user.uut", "active", "true"),
+                        resource.TestCheckResourceAttr("data.btp_directory_user.uut", "family_name", "unknown"), //FIXME should be empty, see NGPBUG-357810
+						resource.TestCheckResourceAttr("data.btp_directory_user.uut", "given_name", "unknown"), //FIXME should be empty, see NGPBUG-357810
```

Comment:
As you can see this is inconsistent to the response for the default IDP. Here the service returns `unknown` as string, in the other case it returns `null`. I've opened a follow up issue in our internal issue tracking system (see NGPBUG-357810).